lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -2100,6 +2105,26 @@ def deploy_vm(self):
                 task = vm.ReconfigVM_Task(vm_custom_spec)
                 self.wait_for_task(task)
 
+            # Set the boot device order
```

Comment:
This will only work with new VM creation. You may want to add functionality to specify boot order while reconfiguring the existing VMs as well.